pcf8563/pcf8563.go

```diff
@@ -0,0 +1,249 @@
+// Package pcf8563 implements a driver for the PCF8563 CMOS Real-Time Clock (RTC)
+//
+// Datasheet: https://www.nxp.com/docs/en/data-sheet/PCF8563.pdf
+//
+
+package pcf8563
+
+import (
+	"time"
+
+	"tinygo.org/x/drivers"
+)
+
+// Device wraps an I2C connection to a PCF8563 device.
+type Device struct {
+	bus     drivers.I2C
+	Address uint16
+}
+
+// New creates a new PCF8563 connection. I2C bus must be already configured.
+func New(i2c drivers.I2C) Device {
+	return Device{
+		bus:     i2c,
+		Address: PCF8563_ADDR,
+	}
+}
+
+// Reset resets the `control and status registers`.
+func (d *Device) Reset() (err error) {
```

Comment:
Should this be renamed to `Configure()`? or should there perhaps be a `Configure()` function added that calls this function?

Reply:
Calling `Reset()`, for example, will initialize Alarm and Timer.

The RTC may be battery-backed even if the microcontroller is powered off. If the RTC is battery-backed, the user will want to use the previous Alarm and Timer, so putting `Reset()` in `Configure()` will not work.
`Reset()` and `Configure()` also have different meanings.

So I think it's good to leave it as it is now. Whether or not to call `Reset()` depends on how to use it.